enhancements/sig-architecture/6-placements/README.md

```diff
@@ -0,0 +1,470 @@
+// 1) Create a ManagedClusterSetBinding in the namespace if they have an RBAC rule to CREATE
+//    on the virtual subresource of `managedclustersets/bind`;
+// 2) Create a service account with name 'placement', and create a ClusterRoleBinding to grant
+//    the CREATE permission on the virtual subresource `managedclustersets/bind` of a ManagedClusterSet
+//    to this service account;
+//
+// A slice of PlacementDecisions with label placement.open-cluster-management.io={placement name}
+// will be created to represent the cluster selected by this placement.
+type Placement struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// Spec defines the attributes of Placement.
+	Spec PlacementSpec `json:"spec"`
+
+	// Status represents the current status of the Placement
+	// +optional
+	Status PlacementStatus `json:"status,omitempty"`
+}
+
+// PlacementSpec defines the attributes of Placement.
+// An empty PlacementSpec selects all clusters from the bound ManagedClusterSets to placement namespace.
+type PlacementSpec struct {
+	// Predicates represent a slice of predicates to select clusters. The predicates are ORed.
+	// +optional
+	Predicates []ClusterPredicate `json:"predicates,omitempty"`
+
+	// Affinity represents the scheduling constraints
+	// +optional
+	Affinity *Affinity `json:"affinity,omitempty"`
+}
+
+// ClusterPredicate represents a predicate to select clusters.
+type ClusterPredicate struct {
+	// ClusterSets represent the ManagedClusterSets from which the clusters are selected.
+	// If the slice is empty, clusters will be selected from the ManagedClusterSets bound to the placement
+	// namespace, otherwise clusters will be selected from the intersection of this slice and the
+	// ManagedClusterSets bound to the placement namespace.
+	// +optional
+	ClusterSets []string `json:"clusterSets,omitempty"`
+
+	// Name represents a selector of clusters by name.
+	// This field is mutually exclusive with other fields in ClusterPredicate.
+	// +optional
+	ClusterName string `json:"clusterName,omitempty"`
+
+	// NumberOfClusters represents the number of clusters to be selected which meet the
+	// requirements in this predicate.
+	// Nil indicates that NumberOfClusters is not set and there is no bound number.
+	// +optional
+	NumberOfClusters *int32 `json:"numberOfClusters,omitempty"`
+
+	// RequiredDuringSchedulingRequiredDuringExecution represents a scheduling strategy for a
+	// cluster selector. If specified, the cluster selector should be mached at both scheduling
+	// time (when the placement decisions are made for the first time) and execution time (After
+	// one or multiple placement decisions are made). It means,
+	// 1) Any cluster, which does not match the selector, should not be selected;
+	// 2) If a selected cluster ceases to match the selector (e.g. due to an update) after the
+	//    placement decisions are made, it will be eventually removed from the placement decisions;
```

Comment:
Any guidance about what should happen when a managecluster is removed from the placementdecisions?  Should whatever was placed there be actively removed?

Reply:
Added a guidance in the comments of `Placement` about what should happen when a managecluster is removed from the placementdecisions.